handmade-example-strings.adoc

```diff
@@ -0,0 +1,681 @@
+.[[D3Example]]<<D3Example,D3>>
+====
+
+++++
+<button class="inspector" onClick="book.main.focus('ui-d3')">Focus Inspector</button>
+<div class="short narrow example" id="ui-d3"></div>
+<br/>
+++++
+
+[source,clojure,role="source"]
+----
+include::src/book/book/ui/d3_example.cljs[]
+----
+
+====
+
+
+;;;;;;;;;;;;;;;;;;;
+
+.[[SampleExample]]<<SampleExample,Sample Example>>
+====
+
+++++
+<button class="inspector" onClick="book.main.focus('example-1')">Focus Inspector</button>
+<div class="short narrow example" id="example-1"></div>
+<br/>
+++++
+
+[source,clojure,role="source"]
+----
+include::src/book/book/example_1.cljs[]
+----
+
+====
+
+
+;;;;;;;;;;;;;;;;;;;
+
+
+
+.[[]]<<,Input Focus and React Refs/Lifecycle>>
+====
+
+++++
+<button class="inspector" onClick="book.main.focus('focus-example')">Focus Inspector</button>
+<div class="short narrow example" id="focus-example"></div>
+<br/>
+++++
+
+[source,clojure,role="source"]
+----
+include::src/book/book/ui/focus_example.cljs[]
+----
+
+====
+
+;;;;;;;;;;;;;;;;;
+
+.[[]]<<,Drawing in a Canvas>>
+====
+
+++++
+<button class="inspector" onClick="book.main.focus('hover-example')">Focus Inspector</button>
+<div class="short narrow example" id="hover-example"></div>
+<br/>
+++++
+
+[source,clojure,role="source"]
+----
+include::src/book/book/ui/hover_example.cljs[]
+----
+
+====
+
+;;;;;;;;;;;;;;;;;
+
+.[[]]<<,Using External React Libraries>>
+====
+
+++++
+<button class="inspector" onClick="book.main.focus('victory-example')">Focus Inspector</button>
+<div class="short narrow example" id="victory-example"></div>
+<br/>
+++++
+
+[source,clojure,role="source"]
+----
+include::src/book/book/ui/victory_example.cljs[]
+----
+
+====
+
+;;;;;;;;;;;;;;;;;
+
+.[[]]<<,Unions to Select Type>>
+====
+
+++++
+<button class="inspector" onClick="book.main.focus('union-example-1')">Focus Inspector</button>
+<div class="short narrow example" id="union-example-1"></div>
+<br/>
+++++
+
+[source,clojure,role="source"]
+----
+include::src/book/book/queries/union_example_1.cljs[]
+----
+
+====
+
+;;;;;;;;;;;;;;;;;
+
+.[[]]<<,UI Blocking>>
+====
+
+++++
+<button class="inspector" onClick="book.main.focus('ui-blocking-example')">Focus Inspector</button>
+<div class="short narrow example" id="ui-blocking-example"></div>
+<br/>
+++++
+
+[source,clojure,role="source"]
+----
+include::src/book/book/server/ui_blocking_example.cljs[]
+----
+
+====
+
+;;;;;;;;;;;;;;;;;
+
+.[[]]<<,Dynamic Query>>
+====
+
+++++
+<button class="inspector" onClick="book.main.focus('dynamic-queries')">Focus Inspector</button>
+<div class="short narrow example" id="dynamic-queries"></div>
+<br/>
+++++
+
+[source,clojure,role="source"]
+----
+include::src/book/book/queries/dynamic_queries.cljs[]
+----
+
+====
+
+;;;;;;;;;;;;;;;;;
+
+.[[]]<<,Dynamic Query Parameters>>
```

Comment:
Yeah, these all need generated anchors and links

Reply:
Yes, I've fixed most parts but I do need some pointers regarding the `PreMerge-*` examples towards the end of the documents regarding the naming of the anchors. If possible please do give a glance at those as well.